### PR TITLE
feat: AJV schema for OCA

### PR DESCRIPTION
**Pull request by rajpalc7**
Accepted incoming changes for static.yml and added validator.yml to resolve the conflict

### CODE REVIEW

File: .github/workflows/static.yml
Comment (amanji) [inline .github/workflows/static.yml:1]: Why was this file removed?

**Review by amanji — CHANGES_REQUESTED** [review]

File: .github/workflows/static.yml
Comment (rajpalc7) [reply .github/workflows/static.yml:1]: @amanji - That file was added in through different branch named validate. I can add the file in this PR if you want

File: .github/workflow/workflows/static.yml
Comment (amanji) [inline .github/workflow/workflows/static.yml:1]: It should still be under `.github/workflows/`

**Review by amanji — CHANGES_REQUESTED** [review]

File: .github/workflow/workflows/static.yml
Comment (rajpalc7) [reply .github/workflow/workflows/static.yml:1]: fixed now

**Comment by amanji** [comment]
Is there any update on this, or can we close until the PR is ready for review again?

**Comment by swcurran** [comment]
@amanji — can you please review. I don’t understand the details.

**Comment by swcurran** [comment]
BTW — what does “AJV” in the title mean?

**Comment by amanji** [comment]
> BTW — what does “AJV” in the title mean?

AJV is the JSON validator library used to check that OCA bundles conform to the OCA for Aries specification

**Pull request closed without merging**